pkgs/by-name/cn/cnsprcy/package.nix

```
@@ -5,30 +5,42 @@
   sqlite,
 }:
 
-rustPlatform.buildRustPackage {
-  pname = "cnsprcy";
-  version = "0.2.0";
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cnspr";
```

Just noticed this. We don't want to change the name of the package, yet.

After you make the change, you can run `git commit --all --amend` and it will be commit to the previous commit which is `cnsprcy: 0.2.0 -> 0.3.2`, then you can just push

```suggestion
  pname = "cnsprcy";
```